### ISSUE TITLE
New problem: Where to post bugs? Which fork is being maintained?

### ISSUE THREAD

**Issue by ammawel** [cakebake/node-red-contrib-alexa-remote-cakebaked#28]
Hello,

I wanted to post an old unresolved bug (https://github.com/cakebake/node-red-contrib-alexa-remote-cakebaked/issues/27) and didn't know where.

In the original 586837r/node-red-contrib-alexa-remote2 it says quite rightly "This fork is DEAD. FFS stop opening issues and READ comment #201!"
There are 45 forks.
In @cakebake's fork, issues are reported but there is no maintenance.
In @bbindreiter's fork, there may be maintenance - he has a pull request in - but no way to post issues.

Is there anything more for this node than updating the module from Apollon77. Where?

Many greetings
Achim

**Comment by bbindreiter** [comment]
I'm allowing issues to be created now on my fork. But please keep in mind that if there are bugs we have to fix them in the correct order. That is, `alexa-remote2` -> `cakebaked` -> `applestrudel`, so everybody benefits from the fixes.

**Comment by peteakalad** [comment]
Sounds good, but does that not mean your fork needs to wait for cakebaked to merge fixes?  

By his own admission cakebaked doesnt use it any more so that order could block fixes into your branch?

(I've switched to your branch appreciate your work, the possible extra feature like extra text commands and prompter fixes are definite cause to do so, thank you)

**Comment by bbindreiter** [comment]
Yes it does. If it's a serious bug I can (and will) of course cherrypick it to avoid the waiting.
